source/geometry_model/spherical_shell.cc

```diff
@@ -210,7 +210,21 @@ namespace aspect
       normal_vector(const typename Triangulation<dim, dim>::face_iterator &face,
                     const Point<dim> &p) const
       {
-        // TODO: Deal with pull back and push forward
+        // We fudge here, but for a good reason. What the function is supposed
```

Comment:
I prefer not to duplicate comments like this and rather refer to the other location.

Reply:
Fair point. Fixed.